IndexedDB/bigint_value.htm

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Values</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+
+<script>
+// BigInt and BigInt wrappers are supported in serialization, per
+// https://github.com/whatwg/html/pull/3480
+// This support allows them to be used as IndexedDB values.
+
+let i = 0;
+function value(value, test) {
+    var t = async_test(document.title + " - " + (i++));
+    t.step(function() {
+        assert_true(test(value),
+                    "condition does not apply to initial value");
+    });
+
+    createdb(t).onupgradeneeded = function(e) {
```

Comment:
wrap the function in t.step_func() so that unexpected exceptions thrown in the async callback are tied to the test, e.g.
```js
createdb(t).onupgradeneeded = t.step_func(e => { ... });
```